3rdparty/pugixml

```diff
@@ -1 +1 @@
-Subproject commit f205aaf6e1c90f1dda943edd4f82922118a85648
+Subproject commit 8b60bbdce428af065f5501fbedd3de3c426073e4
```

Comment:
This must have occurred when trying to switch from zeux/pugixml to RPCS3/pugixml.